x-pack/plugins/security/server/saved_objects/ensure_authorized.ts

```diff
@@ -104,7 +104,10 @@ export async function ensureAuthorized<T extends string>(
 
   // Neither fully nor partially authorized. Bail with error.
   const uniqueUnauthorizedPrivileges = [...missingPrivileges.entries()].reduce(
-    (acc, [, privilegeSet]) => new Set([...acc, ...privilegeSet]),
+    (acc, [, privilegeSet]) => {
+      privilegeSet.forEach((entry) => acc.add(entry));
+      return acc;
+    },
```

Comment:
I'm not sure these are functionally identical. The `Set` was used to enforce uniqueness from my understanding.